### PR TITLE
Update existing database docs with Spawn examples

### PR DESCRIPTION
**Pull request by MatthewFlatt**
Supplements the existing learn more page on existing database setup with some examples using Spawn.

Allows the user to quickly create two instances to represent Production and Development and run through the steps to get comfortable with the process before integrating their existing db.

### CODE REVIEW

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Instead of `we'll use Spawn` I think it would read better if it said `we'll (have|show) examples using Spawn` (the mention of examples gives this a bit more meaning at the beginning here)

Should `instances.` on the end be `databases.` instead, since it's referring to the user's environments which aren't necessarily instances?

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `with the examples` instead of `using the example`

`run the following commands` instead of `run the commands`

`create our Development and Production instances:` instead of `create our environments.`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `PostgreSQL` instead of `Postgres`


`which are used when connecting with Flyway:` instead of `which is used to...` (better to give an example of the only tool that matters here :p)

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Instead of `x`'s, I think it would be better to use the variables mentioned in later examples such as `&lt;DevUsername&gt;`, and also provide a copy for `flyway-container-prod`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: It feels odd to call this a tutorial when there's now an alternative location for Flyway tutorials, so referring to them as examples would be better

`isolated and disposable database instances` instead of `isolated disposable environments`

`in our examples` instead of `for this tutorial`

`section` instead of `heading`

`to follow.` instead of `for the tutorial.`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Would be better to start with `You can download a sample baseline migration` instead of `For the purposes of this tutorial, you can download the baseline script` as it's more succinct

`migration` instead of `file`

End in `:` instead of `.`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `Development database` instead of `development instance`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Extra blank line

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `therefore have nothing to align.` instead of `is of course already aligned.`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: We've made a change recently to hide `n rows affected` from regular output as people found it too spammy, so it'd make sense to delete it here too :)

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `:` instead of `.`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Remove `n rows affected` as in previous comment

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Instead of the italics message, put `...` where necessary in the console output above

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `:` instead of `.`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Just `see` instead of `now see`

`Production` instead of `production`

`to the baseline command` instead of `during the baseline command`

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Extra blank line

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: `If you have followed our examples` instead of `If you have followed through this tutorial using our example`

`fast, isolated copies` instead of `fast isolated copies` (add a comma between them)

`of your own databases, enabling you to` instead of `of your own databases. Enabling you to`

`on a copy of Production before touching the real database` instead of `on a production like copy before the real thing`

`You can find out` instead of `Find out`

**Review by DoodleBobBuffPants — CHANGES_REQUESTED** [review]

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Gone

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done - although used environments as it covers all options

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: Typo in `examples` (`examplies`)

File: documentation/learnmore/existing.md
Comment (DoodleBobBuffPants) [inline documentation/learnmore/existing.md]: The sentence beginning with `Enabling` reads a little odd, and would flow better if it was `your own databases, enabling`

**Review by DoodleBobBuffPants — CHANGES_REQUESTED** [review]

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Oops, good catch!

File: documentation/learnmore/existing.md
Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: I think it becomes a very long sentence then, how about `This enables you`?

Comment (DoodleBobBuffPants) [reply documentation/learnmore/existing.md]: Perfect!

Comment (MatthewFlatt) [reply documentation/learnmore/existing.md]: Done

**Review by DoodleBobBuffPants — APPROVED** [review]

**Comment by MatthewFlatt** [comment]
@DoodleBobBuffPants  Happy for it to be merged/deployed whenever you're ready 👍

**Comment by DoodleBobBuffPants** [comment]
> @DoodleBobBuffPants Happy for it to be merged/deployed whenever you're ready 👍

LGTM, just awaiting a second review :)

**Comment by juliahayward** [comment]
Looks good to me too!

**Pull request merged**
